requirements.txt

```diff
@@ -1,2 +1,2 @@
 asyncssh>=2.10.1
-covalent==0.177.0
+git+https://github.com/AgnostiqHQ/covalent.git@develop#egg=covalent
```

Comment:
are we sure that we want this installing bleeding edge Covalent?

Reply:
I believe the conclusion was to use this until we want to do a release, then we'll switch to the latest stable covalent at that time.

Reply:
@scottwn should I mark this as resolved?

Reply:
@wjcunningham7 can you confirm this?

Reply:
@scottwn I think this may be an ok solution for now as during development we may require the latest changes from covalent and we may not have a pre-release of covalent published yet, furthermore we have situations where we have 1. covalent-aws-plugins installed (depends on covalent) 2. a cloud executor installed (which depends on covalent-aws-plugins and covalent) and in order to avoid version conflicts it may facilitate things if all point to covalent's develop branch